src/features/settings/screen/index.tsx

```diff
@@ -69,6 +74,14 @@ const Settings = () => {
           style={styles.button}
         />
       </View>
+      <View style={styles.bottom}>
+        <Button
+          title="Log Out"
+          disabled={language === 'ar'}
```

Comment:
why this? 🤔

Reply:
just to be able to log in and out for commodity 😅  its not part of the feature

Reply:
oh okay haha